src/ppom-controller.test.ts

```diff
@@ -122,7 +133,7 @@ describe('PPOMController', () => {
         return Promise.resolve('DUMMY_VALUE');
       });
       expect(result).toBe('DUMMY_VALUE');
-      expect(spy).toHaveBeenCalledTimes(5);
+      expect(spy).toHaveBeenCalledTimes(7);
```

Comment:
Hey, curious what caused this number to change for the test ?

Reply:
I'm not sure honestly.